IGI.Lab3/CarPark.WebUI/wwwroot/lib/bootstrap/LICENSE

```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
```

Comment:
@Nightmare-lab I described this issue with lib folder during our meetings.
This folder should absent and libman config should be used.
Asked Sachek Denis about the solution.

Reply:
This issue still not resolved.

Reply:
fix all issues